packages/components/tests/dummy/app/templates/components/form/text-input.hbs

```diff
@@ -267,7 +267,470 @@
+        <F.Label>AWS Account ID</F.Label>
+        <F.HelperText>Copy this ID to your AWS Resource Access Manager to initiate the resource share.</F.HelperText>
+      </Hds::Form::TextInput::Field>
+    '
+  />
+  {{! template-lint-enable no-unbalanced-curlies }}
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">Renders to:</p>
+  <Hds::Form::TextInput::Field @value="036140285924" as |F|>
+    <F.Label>AWS Account ID</F.Label>
+    <F.HelperText>Copy this ID to your AWS Resource Access Manager to initiate the resource share.</F.HelperText>
+  </Hds::Form::TextInput::Field>
+  <p class="dummy-paragraph">When the "helper" text is added, the component automatically adds an
+    <code class="dummy-code">aria-describedby</code>
+    attribute to the input control, associating it with the automatically generated
+    <code class="dummy-code">ID</code>
+    of the helper text element.</p>
+
+  <h5 class="dummy-h5">Extra content in label and helper text</h5>
+  <p class="dummy-paragraph">The
+    <code class="dummy-code">Label</code>
+    and
+    <code class="dummy-code">HelperText</code>
+    contextual components used in the "field" are yielding their content: this means you can pass not just plain text,
+    but also structured content. For example:</p>
+  {{! prettier-ignore-start }}
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::Form::TextInput::Field as |F|>
+        <F.Label>AWS Account ID <Hds::Badge @size="small" @text="Beta" /></F.Label>
+        <F.HelperText>This is an experimental feature (<Hds::Link::Inline @href="#">read more</Hds::Link::Inline>).</F.HelperText>
+      </Hds::Form::TextInput::Field>
+    '
+  />
+  {{! template-lint-enable no-unbalanced-curlies }}
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">Renders to:</p>
+  <Hds::Form::TextInput::Field as |F|>
+    <F.Label>AWS Account ID <Hds::Badge @size="small" @text="Beta" /></F.Label>
+    <F.HelperText>This is an experimental feature (<Hds::Link::Inline @href="#">read more</Hds::Link::Inline>).</F.HelperText>
+  </Hds::Form::TextInput::Field>
+  <p class="dummy-paragraph"><em>Notice: If a link is used within a label, helper text, or error text, it will not be
+      presented as a link to the user with a screen reader; only the text content is read out. Interactive elements in
+      text (associated with the input through aria-describedby) will not be read out as interactive elements to users
+      with screen readers; only the text itself will be read. As such, it is recommended to have a screen reader-only
+      message that informs the user that some help text includes link, and additional keyboard exploration may be
+      required. As such, it is generally preferable to avoid links within help/error text or labels; however, we
+      understand that this may be avoidable in some cases. Please use sparingly until a good known alternative approach
```

Comment:
Shouldn't it be:
```suggestion
      understand that this may not be avoidable in some cases. Please use sparingly until a good known alternative approach
```

Reply:
👍